tests/integration/ceph_multi_cluster_test.go

```diff
@@ -106,46 +115,67 @@ func (mrc *MultiClusterDeploySuite) TestInstallingMultipleRookClusters() {
+		checkIfRookClusterIsInstalled(mrc.Suite, mrc.k8sh, installer.SystemNamespace(mrc.namespace1), mrc.testPVCNamespace, 1)
+		checkIfRookClusterIsHealthy(mrc.Suite, mrc.testClient, mrc.testPVCNamespace)
+	}
 }
 
 // Test Block Store Creation on multiple rook clusters
 func (mrc *MultiClusterDeploySuite) TestBlockStoreOnMultipleRookCluster() {
 	runBlockE2ETestLite(mrc.testClient, mrc.k8sh, mrc.Suite, mrc.namespace1, mrc.op.installer.CephVersion)
 	runBlockE2ETestLite(mrc.testClient, mrc.k8sh, mrc.Suite, mrc.namespace2, mrc.op.installer.CephVersion)
+	if mrc.testPVC {
+		runBlockE2ETestLite(mrc.testClient, mrc.k8sh, mrc.Suite, mrc.testPVCNamespace, mrc.op.installer.CephVersion)
+	}
 }
 
 // Test Filesystem Creation on multiple rook clusters
 func (mrc *MultiClusterDeploySuite) TestFileStoreOnMultiRookCluster() {
 	runFileE2ETestLite(mrc.testClient, mrc.k8sh, mrc.Suite, mrc.namespace1, "test-fs-1")
 	runFileE2ETestLite(mrc.testClient, mrc.k8sh, mrc.Suite, mrc.namespace2, "test-fs-2")
+	if mrc.testPVC {
+		runFileE2ETestLite(mrc.testClient, mrc.k8sh, mrc.Suite, mrc.testPVCNamespace, "test-fs-3")
+	}
 }
 
 // Test Object Store Creation on multiple rook clusters
 func (mrc *MultiClusterDeploySuite) TestObjectStoreOnMultiRookCluster() {
 	runObjectE2ETestLite(mrc.testClient, mrc.k8sh, mrc.Suite, mrc.namespace1, "default-c1", 2)
 	runObjectE2ETestLite(mrc.testClient, mrc.k8sh, mrc.Suite, mrc.namespace2, "default-c2", 1)
+	if mrc.testPVC {
+		runObjectE2ETestLite(mrc.testClient, mrc.k8sh, mrc.Suite, mrc.testPVCNamespace, "default-c3", 1)
+	}
 }
 
 // MCTestOperations struct for handling panic and test suite tear down
 type MCTestOperations struct {
-	installer       *installer.CephInstaller
-	kh              *utils.K8sHelper
-	T               func() *testing.T
-	namespace1      string
-	namespace2      string
-	systemNamespace string
+	installer        *installer.CephInstaller
+	kh               *utils.K8sHelper
+	T                func() *testing.T
+	namespace1       string
+	namespace2       string
+	testPVC          bool
+	testPVCNamespace string
+	systemNamespace  string
 }
 
 // NewMCTestOperations creates new instance of TestCluster struct
-func NewMCTestOperations(t func() *testing.T, namespace1 string, namespace2 string) (*MCTestOperations, *utils.K8sHelper) {
+func NewMCTestOperations(t func() *testing.T, namespace1 string, namespace2 string, testPVCNamespace string) (*MCTestOperations, *utils.K8sHelper) {
 
 	kh, err := utils.CreateK8sHelper(t)
 	require.NoError(t(), err)
 	checkIfShouldRunForMinimalTestMatrix(t, kh, multiClusterMinimalTestVersion)
 
 	i := installer.NewCephInstaller(t, kh.Clientset, false, installer.VersionMaster, installer.NautilusVersion)
 
-	op := &MCTestOperations{i, kh, t, namespace1, namespace2, installer.SystemNamespace(namespace1)}
+	op := &MCTestOperations{i, kh, t, namespace1, namespace2, false, testPVCNamespace, installer.SystemNamespace(namespace1)}
+	p := kh.IsStorageClassPresent("gp2")
```

Comment:
In the CI log I see that `usePVCs=false`. We must need to do something to enable gp2 in the CI. This may be the trickiest part of enabling this test...

Reply:
don't have any clue how can we enable it. can we create it by ourselves?